rca/project_styleguide/templates/patterns/organisms/news/news_api_content.html

```diff
@@ -16,8 +16,8 @@
         {% endif %}
 
         {% if forloop.last and not forloop.first %}
-                <a class="news__view-all link link--primary link--link body body--two" href="{{ news_view_all.link|default:'/news-and-events/' }}">
-                    <span class="link__label">{{ news_view_all.title|default:'See all news & events' }}</span>
+                <a class="news__view-all link link--primary link--link body body--two" href="{{ page.news_view_all.link|default:'/news-and-events/' }}">
```

Comment:
Question - are we confident that the default /news-and-events/ route will always resolve? A default page selected on a settings model seems more robust.

Reply:
Given the project convention for using URLFields I'd say this is fine then